test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -0,0 +1,62 @@
+
+task ValidateOptimus {
+      File bam
+      File matrix
+      File matrix_summary
+      File picard_metrics
+      Int required_disk = ceil((size(bam, "G") + size(matrix, "G") + size(matrix_summary, "G") + size(picard_metrics, "G")) * 1.1)
+
+      String expected_bam_hash
+      String expected_matrix_hash
+      String expected_matrix_summary_hash
+      String expected_picard_metrics_hash
+      # # these are also inputs; lets leave them alone until flattening is in WDL
```

Comment:
Do you want to make a ticket or something so we know this should be addressed at some point?  I leave it up to you if you want to remove the commented out code or not.